### PR TITLE
Fixed navbar issues on thin screens

### LINKED ISSUE #816: Menu tab in navbar is unusable on smaller windowed screens
The navbar automatically hides the "retake tutorial" "faq" and "sign in" buttons into a collapsible pane called "menu" when the browser window is below a certain width. Unfortunately, it is near impossible to navigate menu; the options jump to the left side of the screen, appear in a very short scrollable dropdown, and have a transparent background until hovered:

image

This is also the case on the live site:
image

Since our users may use these smaller resolutions, we should probably either disable the "menu" dropdown and leave the navbar links in place on smaller res screens, or fix the positioning of the items in this dropdown.

### PR DESCRIPTION
**Pull request by sbower213**
Resolves #816. There were some issues with the appearance and functionality of the navbar if a user was using a browser window below a certain width. Additionally, some CSS changes were inadvertently rolled back in a previous branch (my guess is that this is from when I added kotaro's old async login code) so I reapplied those.

### CODE REVIEW

**Comment by maddalihanumateja** [comment]
The navbar resizing seems to be fine. You seem to have worked on multiple issues in this pull request which made testing this request a bit confusing. Could you list out the issues individually? Did you add prompts for incorrect password on signin and signup in this request?

I'm getting an incorrect email address or password prompt when I try to sign up. It doesn't seem to be too informative so I was wondering what I was doing incorrectly.

image

**Comment by manaswis** [comment]
I think he branched off of the branch that had another PR's (#818) changes that he submitted previously. So once those changes are merged with develop, this branch shouldn't show them explicitly in this branch. @sbower213 In the future, please don't do this type of branching unless the issues are related.

**Comment by manaswis** [comment]
@maddalihanumateja Don't merge this branch just yet. PR #818 needs to be merged first since few other things have changed related to #818 since he worked on that PR.

**Comment by sbower213** [comment]
Sorry, I definitely meant to branch off of develop! I'll take more care to make sure I do that in the future.

**Comment by manaswis** [comment]
@sbower213 Whenever you feel better, could you fix the conflicts? Then this can be merged as it is tested by @maddalihanumateja, correct?

**Comment by sbower213** [comment]
OK, I just rebased off develop and fixed the merge conflicts. This should be ready for testing.

**Comment by jonfroehlich** [comment]
Can we test this and get it into production? Who is doing the PR review and test?

File: app/views/navbar.scala.html [hunk 1]
```diff
@@ -245,19 +231,44 @@ <h4 class="modal-title" id="sign-up-label">Sign up</h4>
                     }
                 });
 
-                $("[data-hide]").on("click", function(){
-                    $("." + $(this).attr("data-hide")).hide();
-                    // -or-, see below
-                    // $(this).closest("." + $(this).attr("data-hide")).hide();
+                //set tooltip alerts for when users do not enter all sign in / sign up fields
```
Comment (manaswis) [inline app/views/navbar.scala.html]: This is old code. @sbower213 can you make sure this file has the proper email validation that was updated in https://github.com/ProjectSidewalk/SidewalkWebpage/pull/883/files

**Comment by sbower213** [comment]
New pull request in #1020

**Pull request closed without merging**